product_docs/docs/eprs/7/02_overview/04_design_replication_system/03_restrictions_on_replicated_database_objects.mdx

```diff
@@ -132,3 +132,55 @@ You can include Postgres tables containing the built-in range types in the publi
 -   If a publication table of the primary definition node in a multi-master replication system contains any built-in range types, then you can add a database as a primary node only if the database server of this intended primary node is Postgres version 9.2 or later.
 
 Custom range types constructed with the `CREATE TYPE AS RANGE` command aren't supported in Replication Server.
+
+## Limitations during re-snapshot for views
+
+As part of the re-snapshot process, when Replication Server drops constraints from the table’s column that are the part of view objects using `CASCADE` option, the views might also get dropped. Here is a simplified example of the limitation:
```

Comment:
@ccestes I suggest updating this paragraph with the suggested changes I recommended in a recent comment on XDB-1884.